haproxy/default.toml

```diff
@@ -1,6 +1,7 @@
 maxconn = 32
 mode = "http"
 bind = "*:80"
+httpchk = "GET /favicon.ico"
```

Comment:
Is this a sane default? So many services never implement a favicon.ico.

From the docs:

> It defaults to " / " which is accessible by default on almost any server, but may be changed to any other URI